docker-compose.yml

```diff
@@ -10,4 +10,4 @@ services:
     stdin_open: true
 
   redis:
-    image: redis:3.0.5
+    image: redis:6.2.6
```

Comment:
Die ganzen Aktualisierungen würde man in dem PR nicht erwarten. Pragmatischer Zwischenweg wäre es in die Story und in den Commit dazuzuschreiben.

Reply:
stimmt, aber das Dockerfile war so alt, das ich aktualisieren musste. Und das neue redis Gem ist mit der 3er Version nicht mehr gelaufen. Und so sind die Specs auch näher an Production, da es die gleiche Version von Redis ist, wie auf atalanda

Reply:
Das die Änderung eine gute Änderung ist, steht eh außer Frage :+1: